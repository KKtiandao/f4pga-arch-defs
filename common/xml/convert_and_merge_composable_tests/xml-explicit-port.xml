<?xml version="1.0"?>
<xml>
  <port name="port-name" />
  <port name="port-name" bit="8" />
  <port name="port-name" bit_msb="8" bit_lsb="4" />
</xml>
