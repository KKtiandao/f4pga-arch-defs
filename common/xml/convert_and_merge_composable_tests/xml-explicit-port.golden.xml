<?xml version="1.0"?>
<xml>
port-name
port-name[8]
port-name[8:4]
</xml>
